### PR TITLE
Change GreenGuard to Draco

### PR DESCRIPTION
**Pull request by sarahmish**
move everything to `draco`.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+docker run -ti -p8888:8888 signalsdev/draco:latest
 ```
 
-This will start a Jupyter Notebook instance on your computer already configured to use GreenGuard.
+This will start a Jupyter Notebook instance on your computer already configured to use Draco.
 You can access it by pointing your browser at http://127.0.0.1:8888
 
 Further details about the usage of this image can be found [here](
-https://hub.docker.com/repository/docker/signalsdev/greenguard).
+https://hub.docker.com/repository/docker/signalsdev/draco).
```
Comment (dyuliu) [inline docker/README.md]: It seems that draco is under signalsdev?

We should also check orion and other libraries as well if they use `docker`

File: docker/README.md [hunk 2]
```diff
@@ -29,11 +29,11 @@ For this example, we are assuming that the yaml file is named `greegunard-pod.ym
...
 kind: Pod
 metadata:
-  name: greenguard
+  name: draco
 spec:
   containers:
-  - name: greenguard
-    image: signalsdev/greenguard:latest
+  - name: draco
+    image: signalsdev/draco:latest
```
Comment (dyuliu) [inline docker/README.md]: the same here. signalsdev -> sinteldev

File: docs/advanced_usage/concepts.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ hyperparameters for a Template. Hence, Pipelines:
   hyperparameters of their template.
 
 A pipeline can be fitted and evaluated directly using [MLBlocks](
-https://hdi-project.github.io/MLBlocks), or using the **GreenGuardPipeline**.
+https://MLBazaar.github.io/MLBlocks), or using the **DracoPipeline**.
```
Comment (dyuliu) [inline docs/advanced_usage/concepts.md:37]: Perhaps give a link to the DracoPipeline document or Code file?

**Review by dyuliu — APPROVED** [review]

File: docker/README.md (hunk 1 above)
Comment (sarahmish) [reply docker/README.md]: For Orion, you can compile a docker instance but we did not push a container to the hub like GreenGuard. We need to update this actually.

File: docs/advanced_usage/concepts.md (hunk 3 above)
Comment (sarahmish) [reply docs/advanced_usage/concepts.md:37]: I will mark this for when we have good documentation on Draco pipelines

File: setup.cfg [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.3.1.dev0
+current_version = 0.0.1.dev0
```
Comment (frances-h) [inline setup.cfg:2]: Is there a reason we changed the version here, and should there be a corresponding change in `setup.py`? I tested this out locally and it seems like bumpversion complains if this version doesn't match the version in `setup.py`.

Comment (sarahmish) [reply setup.cfg:2]: Good catch, I'll revert it there as well.

**Pull request merged**